.travis.yml

```diff
@@ -4,7 +4,7 @@ language: python
 
 cache: pip
 
-python: ["2.7", "3.2", "3.3", "3.4", "3.5", "3.6", "pypy"]
+python: ["2.7", "3.4", "3.5", "3.6", "pypy"]
```

Comment:
Seems 3.3 and 3.2 are not archives anymore? Only removed to get CI to pass green.
https://github.com/travis-ci/travis-ci/issues/9133#issuecomment-506303002

Reply:
@UberMouse thoughts?

Reply:
@samuel-holt ? anybody?

Reply:
This seems like quite a common issue. What was the actual error in the CI?

Reply:
Have you looked at a workaround like this one: https://github.com/aws/base64io-python/issues/4#issuecomment-412696588?

Reply:
Cool, I'll check it out.

Reply:
Got it fixed. The older Ubuntu version happens to have support for Python 3.2 & 3.3.
```
    - python: 3.2
      dist: trusty
```

Reply:
Awesome! Good find 👍

Reply:
Thanks

Reply:
Now you've fixed the issue causing CI to fail, can you re-instate versions 3.2 and 3.3?

Reply:
Yep, confused me too. The `matrix` will run those versions, even though they are not in the `python: []` list.

UberMouse did the [same thing here](https://github.com/MindscapeHQ/raygun4py/commit/2e7149094bc3e850783256b170bc91aa9eda9fe9), he removed 3.7 from the list but also added it to matrix.

I even tried it with both `matrix` and inside `python: []` and it duplicates 3.2 and 3.3 builds. One of the 3.2 builds succeeds, and the other fails.
<img width="1156" alt="Screen Shot 2019-09-04 at 10 50 01 PM" src="https://user-images.githubusercontent.com/14981893/64308367-fffe9900-cf66-11e9-80b4-c58b67dd8dd3.png">